### ISSUE TITLE
Code description

### ISSUE THREAD

**Issue by ronanyeah** [foundersandcoders/express-workshop#6]
In the [hello world](https://github.com/PiotrBerebecki/express-workshop#express-hello-world) you've written out some great code explanations that might be easier to digest if they were comments within the code.

**Comment by akinsho** [comment]
@ronanyeah I agree re. comments but @PiotrBerebecki is currently away so I'm handling the issues not sure I can go through and add comments in time but will try to get them in before this is done in nazareth and Gaza
